### PR TITLE
closes #29: Add OpenCensus to model.

### LINKED ISSUE #29: Add OpenCensus to the model

### PR DESCRIPTION
**Pull request by tzwickl**

### CODE REVIEW

File: model/components/opencensus.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        - python
+        - ruby
+      - aspects:
+        - tracing
+        - metrics-app
+        - metrics-process
+    connections:
+        - prometheus-server
+        - stackdriver
+        - zipkin-server
```
Comment (AlexanderWert) [inline model/components/opencensus.yml]: As far as I know, Stackdriver is not open-source. So let's omit it here.

File: model/components/opencensus.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+components:
+
+  # opencensus instrumentation toolkit
+  - id: opencensus
+    name: OpenCensus
+    description: A single distribution of libraries that automatically collects traces and metrics from your app, displays them locally, and sends them to any analysis tool.
```
Comment (AlexanderWert) [inline model/components/opencensus.yml]: "... and sends them to any analysis tool" I would rather write something like: "... and provides means to write to different analysis tools"

File: model/components/opencensus.yml (hunk 1 above)
Comment (tzwickl) [reply model/components/opencensus.yml]: yes you are right it free to use but not OpenSource :)

File: model/components/opencensus.yml (hunk 2 above)
Comment (tzwickl) [reply model/components/opencensus.yml]: ok changed it ;) Was the official text from their website ^^

**Pull request merged**